patches/crypto-unexportable_key_metrics.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/crypto/unexportable_key_metrics.cc b/crypto/unexportable_key_metrics.cc
+index 4217c2aaf95126d6606fdc3cb568f6a1030002f1..de7c926d46fab8efcf92ad4370f7aaa07b8c91bb 100644
+--- a/crypto/unexportable_key_metrics.cc
++++ b/crypto/unexportable_key_metrics.cc
+@@ -95,6 +95,7 @@ void MeasureTpmOperationsInternal() {
+   if (algo) {
+     switch (*algo) {
+       case SignatureVerifier::SignatureAlgorithm::ECDSA_SHA256:
++      case SignatureVerifier::SignatureAlgorithm::ECDSA_SHA384:
```

Comment:
`#define RSA_PKCS1_SHA1 ECDSA_SHA384: case SignatureVerifier::SignatureAlgorithm::ECDSA_SHA256`

Reply:
even though these should all be convertible to chromium_src, I still have concerns about this because there are a lot of different places this needs to go and it could be missed in places where there is maybe if/else instead of case statement. Can we not use a different algorithm that is already supported in chromium? Or alternatively can we just just directly implement the signature verification instead of adding it to the enum in signature_verifier.h?